cmd/store/store.go

```diff
@@ -321,19 +413,116 @@ func (s *Store) ListEntries(name ...string) ([]string, error) {
 	return entries, nil
 }
 
-func (s *Store) StoreClaim(data []byte) error {
+func (s *Store) ClaimType(data []byte) (*jwt.ClaimType, error) {
 	// Decode the jwt to figure out where it goes
 	gc, err := jwt.DecodeGeneric(string(data))
 	if err != nil {
-		return fmt.Errorf("invalid jwt: %v", err)
+		return nil, fmt.Errorf("invalid jwt: %v", err)
 	}
 	if gc.Name == "" {
-		return errors.New("jwt claim doesn't have a name")
+		return nil, errors.New("jwt claim doesn't have a name")
+	}
+	return &gc.Type, nil
+}
+
+func (s *Store) syncRemoteAccount(u string) (int, []byte, error) {
```

Comment:
i guess you have to take a URL here, but it feels weird that the store doesn't even know the account being sync'd as a result